src/main/scala/io/iohk/ethereum/network/TestSocketHandshaker.scala

```diff
@@ -55,22 +54,45 @@ object TestSocketHandshaker {
       nodeId = ByteString(nodeId))
 
     sendMessage(helloMsg, frameCodec, out)
-
     val remoteHello = readAtLeastOneMessage(frameCodec, inp).head.asInstanceOf[Hello]
 
     while (true) {
       val msgs = readAtLeastOneMessage(frameCodec, inp)
-      msgs.foreach { m => println("\n Received message: " + m) }
+      msgs.foreach { m =>
+        //println("\n Received message: " + m)
```

Comment:
Minor detail: Maybe you forgot about this comment, maybe this should be uncommented or deleted?